features/grpc/exporter/pom.xml

```diff
@@ -161,5 +160,155 @@
             <artifactId>protobuf-java</artifactId>
             <version>${protobufVersion}</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opennms.core.test-api</groupId>
+            <artifactId>org.opennms.core.test-api.lib</artifactId>
+        </dependency>
+
+        <!-- gRPC Dependencies -->
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-netty-shaded</artifactId>
+            <version>${grpcVersion}</version> <!-- Use your defined grpcVersion -->
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-protobuf</artifactId>
+            <version>${grpcVersion}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-stub</artifactId>
+            <version>${grpcVersion}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Although initially you can do trail and error to see which dependencies are needed, you should remove those dependencies that are not needed once you get tests working